core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/AutoscaleStackV4Response.java

```diff
@@ -72,6 +72,9 @@
     @ApiModelProperty(StackModelDescription.TYPE)
     private StackType stackType;
 
+    @ApiModelProperty(StackModelDescription.ENVIRONMENT_CRN)
+    private String environmentCrn;
+
```

Comment:
is it required to add environmentCrn to the API also? We are using it for internal logic only as I see